test/unit/tour.spec.js

```diff
@@ -446,16 +451,17 @@ describe('Tour | Top-Level Class', function() {
       const step = instance.addStep({
         id: 'test',
         title: 'This is a test step for our tour',
-        popperOptions: {
-          modifiers: [{ name: 'foo', options: 'bar' }]
-        }
+        floatingUIOptions: {
+          middleware: [{name: 'foo', options: 'bar', fn: (args) => args}],
+        },
       });
 
       instance.start();
 
-      const popperOptions = setupTooltip(step);
+      const floatingUIOptions = setupTooltip(step);
 
-      expect(popperOptions.modifiers.length).toBe(5);
+      // @todo should be 3 here.
```

Comment:
need to dig in what's going on here. should be 3 middleware, the default 2 + the `foo` middleware